src/CoinTossGame.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Scanner;
+
+public class CoinTossGame {
+    private static Scanner scanner = new Scanner(System.in);
+    public static void main(String[] args) {
+        CoinTossGame  game = new CoinTossGame();
```

Comment:
You can make askPlayersName(), printMessageForUser() and other methods in this class as static and use them in main method, so you don't have to instantiate a new object here :)